config.toml

```diff
@@ -104,8 +104,14 @@ URL = "/"
 weight = 1
 
 [[Languages.en.menu.main]]
-name = "Get Wasmcloud"
+name = "Try Now"
+URL = "https://www.katacoda.com/wasmcloud#"
 weight = 2
+
+
+[[Languages.en.menu.main]]
+name = "Get Wasmcloud"
+weight = 3
```

Comment:
I thought we settled on the following structure, renaming `Get Wasmcloud` to `Getting Started`:
```
Getting Started
- Try wasmCloud
- Install
- GitHub
```

Reply:
I looked at it both ways and liked the stronger call to action here.  Let's see what kind of traction we get with it here; we can move it if not.

Reply:
Then perhaps the only change needed is changing capitalization of `Wasmcloud` to `wasmCloud`, unless it should be capitalized in this scenario.

Reply:
@LiamRandall ^